scripts/objectstore/copy_files_to_irods.py

```diff
@@ -0,0 +1,94 @@
+import os
+import subprocess
+import sys
+import time
+sys.path.insert(0, '../../packages/objectstore/')
+
+from psycopg2 import connect, sql
+
+from galaxy.util import directory_hash_id
+
+
+def copy_file_to_irods(dataset_id, objectstore_id="", galaxy_dir="/Users/kxk302/workspace/galaxy", object_store_info={"files1" : "database/files1", "files2" : "database/files2"}, irods_home="/tempZone/home/rods"):
```

Comment:
Can remove objectstore_id from list of input parameters and down in sql_statement.